src/Wrapper.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rubix\ML;
+
+/**
+ * Wrapper
+ *
+ * @category    Machine Learning
+ * @package     Rubix/ML
+ * @author      Andrew DalPino
```

Comment:
Shouldn't this be your name and not my name?

Reply:
Done 👍